assembl/static2/js/app/components/common/medias.jsx

```diff
@@ -0,0 +1,20 @@
+// @flow
+import React, { Fragment } from 'react';
+
+import { videoUtilities } from '../../utils/videoUtilities';
+
+export type Props = {
+  path: string
+};
+
+export const Medias = ({ path }: Props) => (
+  <Fragment>
+    {videoUtilities.pathIsVideoFile(path) ? (
+      <video src={path} controls preload="none" width="100%" /> // eslint-disable-line
```

Comment:
Why eslint-disable-line :) ?

Reply:
The error message is: "Média elements such as a <audio> ans <video> must have a <track> for captions". Any ideas?

Reply:
Ok forget it, this is about adding text info about the video for deaf users, you would have to add a <track> element with a dybnamic src set by the video uploader I guess... so clearly out of scope

Reply:
c'est pour l'accessibilité